### ISSUE TITLE
Evaluation Problem

### ISSUE THREAD

**Issue by selmiss** [ContextualAI/gritlm#11]
Dose your evaluation code support data parallelism? I couldn't find any module to mteb evaluation that use DP or DDP

**Comment by selmiss** [comment]
Thank you. And by the code we can only use DP or PP, can't mix using these two?

**Comment by Muennighoff** [comment]
> Thank you. And by the code we can only use DP or PP, can't mix using these two?

Yeah only one or the other for mteb evaluation; I'm not sure if transformers supports mixing them
